src/routes/home/components/GameListItem.vue

```diff
@@ -81,6 +81,10 @@ export default {
       type: Number,
       required: true,
     },
+    players: {
+      type: Array,
+      required: true,
+    },
```

Comment:
We should also remove `numPlayers` as a prop and replace it with a computed property that returns the length of the `players` array. Or alternatively remove it altogether and just check `this.players.length` wherever we need the numPlayers